src/features/auth/sign-in/components/user-auth-form.tsx

```
@@ -45,14 +46,26 @@
+      const response = await apiClient.post('auth/login', data)
+      // Assuming the response returns an object with { user, token }
+      const { user, token } = response.data
 
-    setTimeout(() => {
+      // Update auth store
+      setUser(user)
+      setAccessToken(token)
+
+      // Redirect to the authenticated part of your app
+      navigate({ to: '/chats', replace: true })
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)